README.md

```diff
@@ -26,6 +36,53 @@ The MWI creates a score for each zip code between 0 and 100, such that:
 
 The MWI was developed with the mental health status of Black Americans in mind. We selected Black Americans as a priority population in order to center at the margins and avoid creating an index that is focused to the 'average community.'  We believe that focusing on Black Americans in this way allows all groups experiencing disparities to benefit because none of us are well until all of us are well. We also recognize the need to identify, recognize, and adapt the MWI for additional priority populations.
 
+# Instructions to Create Your Own MWI
+
+To create your own Mental Wellness Index, you must be running the Mental Wellness Index Tool on your local computer in order to protect your data. Follow the instructions below to create your own MWI for your community below by adjusting weights and/or adding your own data and metadata.
+
+1. Download R and RStudio. Download a modern browser (Firefox, Chrome, Edge, etc.) and make that your default browser if you haven't already.
```

Comment:
Add hyperlink to R and RStudio? 

https://www.rstudio.com/products/rstudio/download/
https://www.r-project.org/

May also want to mention somewhere that all of this is free.

Reply:
Whoops! Had these in the app, but not the README. Added these. Also added the free bit.